app/panel/utils/utils.js

```diff
@@ -171,3 +171,57 @@ export function doXHR(method, url, query) {
 		xhr.send(query);
 	});
 }
+/**
+ * Determine if user is a subscriber
+ * @memberOf PanelUtils
+ * @param  {array} userScopes 	array of scope strings
+ * @return {boolean}
+ */
+export function isSubscriber(user) {
+	if (!user || !user.scopes) { return false; }
+	const userScopes = user.scopes;
+
+	// check scopes
+	if (userScopes.indexOf('god') >= 0) { return true; }
+	if (userScopes.indexOf('subscription:supporter') >= 0) { return true; }
+
+	return false;
+}
```

Comment:
@zarembsky we should be using `Account.hasScopesUnverified` here.

Reply:
Account.hasScopesUnverified is a background API, which uses conf. function